README.md

```diff
@@ -111,7 +111,13 @@ a 503.
 In order to be able to securize your virtual host, you have to create a file named as its equivalent VIRTUAL_HOST variable on directory
 /etc/nginx/htpasswd/$VIRTUAL_HOST
 
-    $ docker run -d -p 80:80 -p 443:443 -v /path/to/htpasswd:/etc/nginx/htpasswd -v /path/to/certs:/etc/nginx/certs -v /var/run/docker.sock:/tmp/docker.sock jwilder/nginx-proxy
+```
```

Comment:
Oh, that's a shame 😢

Not sure about dropping the `$`. I agree it's easier to copy/pasta but it's a bit of a convention to include it (at least in the docker docs)